CHANGELOG.md

```diff
@@ -1,10 +1,16 @@
-#Change Log
+# Change Log
 This project adheres to [Semantic Versioning](http://semver.org/).
 
 This CHANGELOG follows the format listed at [Keep A Changelog](http://keepachangelog.com/)
 
 ## [Unreleased]
 
+### Breaking Change
+- bumped requirement of `sensu-plugin` to 2.x (@majormoses)
```

Comment:
I'd add a "See the `sensu-plugin` 2.0 CHANGELOG for details" link here. Now that we've linked the blog this will help people be aware of the breaking changes.